.github/workflows/ci.yml

```diff
@@ -10,7 +10,8 @@ concurrency:
 jobs:
   test:
     name: Test Go
-    runs-on: ubuntu-latest
+    runs-on:
+      group: ftl
```

Comment:
Does this need to specify the `runner` name and the `group`? Or is just the `group` enough here? In other setups I've seen something like:

```yml
runs-on: my-cool-custom-runner
```

Reply:
Group is _supposedly_ [sufficient](https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/running-jobs-on-larger-runners#using-groups-to-control-where-jobs-are-run).